pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -416,6 +416,22 @@ in
     ];
   };
 
+  pg_query = attrs:
+    let
+      libpg_query = fetchurl {
+        url = "https://codeload.github.com/lfittl/libpg_query/tar.gz/10-1.0.1";
```

Comment:
This would pin the same version of libpg_query for all the released of the pg_query gem. Not sure what's the best option there...

Ideally we would know what version the gem wants to fetch and the ask the user to provide the right sha256 for that version.